dozer-types/src/models/api_endpoint.rs

```diff
@@ -20,6 +82,9 @@ pub struct ApiEndpoint {
     pub path: String,
     #[prost(message, tag = "4")]
     pub index: Option<ApiIndex>,
+    #[prost(message, tag = "5")]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub conflict_resolution: Option<ConflictResolution>,
```

Comment:
We can use `serde(default)` to achieve the same configuration experience without all the `Option`s?

Reply:
Prost `message` expects type to implement `as_ref` and `get_or_insert` methods, so in this case we can continue using `Option`

Reply:
Is it possible to `derive(::prost::Message)` for `ConflictResolution`?

Reply:
Sorry, just saw you've done that.